cosmo_tester/test_suites/image_based_tests/cfy_manager_test.py

```diff
@@ -98,6 +120,31 @@ def test_cfy_manager_configure(image_based_manager, logger, test_config):
         )
 
 
+def test_cfy_manager_upgrade(manager_5_1_0, ssh_key, logger, test_config):
+    example = get_example_deployment(
+        manager_5_1_0, ssh_key, logger, 'manager_upgrade', test_config)
+    example.upload_and_verify_install()
+    validate_agents(manager_5_1_0, example.tenant)
+
+    logger.info('Installing new RPM')
+    manager_5_1_0.run_command('yum install -y {rpm}'.format(
+        rpm=test_config['upgrade']['upgrade_rpm_path']), use_sudo=True)
+    logger.info('Upgrading manager')
+    manager_5_1_0.run_command('cfy_manager upgrade -v')
+
+    _validate_manager_status(manager_5_1_0)
+    validate_agents(manager_5_1_0, example.tenant)
```

Comment:
After this you should probably have it run example.uninstall() as this gives an even more clear indication that things are indeed fully functional.

Reply:
Correct, I forgot about it. Thanks!